enhancements/release/improving-ci-signal.md

```diff
@@ -144,15 +144,17 @@ results in a JobFooAggregate Job that can pass or fail based on historical crite
 
 #### Ok, What then? We’re already busted
 
-Fast-forward to a point where we actually built this mechanism. We’ve now got a broken payload. It’s good that we know
-it’s broken, but it’s still broken.
+Once we know we have a broken payload, TRT must revert the breaking change. Regardless if a likely fix is identified,
+the revert must land first. A clean revert is nearly always guaranteed to fix the problem immediately, while a proposed
+fix PR is not. Additionally, it is a negligible amount of additional effort to build your fix on top of unreverting the
+revert.
 
 ### Quick Revert
 
-TRT is expected to watch payload promotion for master (not historical releases). Given a broken payload that TRT
-detected quickly, provide a way to quickly and definitively identify the regressing change. Merge a nearly immediate
-revert after proving the revert resolves the problem. Provide a way for the un-revert to prove that it is no longer
-regressing before it merges.
+TRT is expected to watch payload promotion for the main branch (not historical releases). Given a broken payload that
+TRT detected quickly, quickly and definitively identify the regressing change. Once identified, merge an immediate
```

Comment:
This replaces "proving the revert resolves the problem" with the more ambiguous "definitively identify".  Is is worth explicitly calling out the ambiguous cases explicitly?  Maybe language like:

> When several recent changes are all suspicious, immediately revert all of them.  Testing on an un-revert PR can demonstrate when a reverted change was actually unrelated, and the un-revert can then land without overly delaying the component team.

Or something?

Reply:
> This replaces "proving the revert resolves the problem" with the more ambiguous "definitively identify". Is is worth explicitly calling out the ambiguous cases explicitly? Maybe language like:

Based on the last six months or so of experience, many (most?) cases are fairly straight-forward to identify and in cases where it's not clear, the steps towards definitive identification have varied.  I think gives a clear sentiment, without restricting the tools.